spec/factories/observations.rb

```diff
@@ -1,7 +1,40 @@
 FactoryBot.define do
   factory :observation do
     user
+    taxon
     license { Observation::CC_BY }
     description { Faker::Lorem.sentence }
+    uri { Faker::Internet.url }
+    observed_on_string { 'yesterday at noon' }
+    observed_on { Date.yesterday }
+    time_observed_at { Date.yesterday.noon }
+    time_zone { 'UTC' }
+    zic_time_zone { 'UTC' }
+  end
+
+  trait :without_times do
+    observed_on { nil }
+    observed_on_string { nil }
+    time_observed_at { nil }
+    time_zone { nil }
+    zic_time_zone { nil }
+  end
+
+  trait :research_grade do
+    taxon { build :taxon, :as_species }
+    latitude { 1 }
+    longitude { 1 }
+    quality_grade { Observation::RESEARCH_GRADE }
+    identifications { [association(:identification)] }
```

Comment:
I don't think this will actually make a Research Grade observation, right? Seems like this is just going to make *an* identification associated with the observation, not an identification matching the identification the observer chose. And no media.

I don't think this needs to block the merge, but something to keep an eye on.  We might want to name this something to indicate that it's only making something that's sort of an RG obs, maybe only suitable for unit testing, and `make_research_grade_observation` should be used for more integrative tests.

Reply:
Ahh, I see, absolutely something to correct early on! Particularly before it's used too much and giving a false sense of intent. I tried to address here: https://github.com/inaturalist/inaturalist/pull/3189 . Hopefully this can stub out (or create) something that qualifies as research grade.